src/core/packages/chrome/layout/core-chrome-layout-components/layout.types.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import React from 'react';
+
+export interface LayoutStyleArgs {
+  bannerHeight: number;
```

Comment:
```suggestion
  /** The height of the top-most banner */
  bannerHeight: number;
```

Reply:
I am adding the comments, but I am not sure how valuable they are? aren't they just repeat what the prop name or component name says? e.g. bannerHeight === /** The height of the top-most banner */